doc/classes/Object.xml

```diff
@@ -137,12 +141,14 @@
 			<argument index="0" name="signal" type="String">
 			</argument>
 			<description>
+				Calls all methods connected to the [code]signal[/code].
```

Comment:
This is not a good description of what a signal is. Emitting a signal does not call the connected methods. That is the responsibility of any connected objects. The emitting object has no knowledge of any connections - there may not even be any. The signal is still emitted.

Emits the given [code]signal[/code].